docs/languages.md

```diff
@@ -0,0 +1,55 @@
+# Languages
+
+## Downloading
+
+### Through Steam
+
+1. If you own the game on Steam, you can download the language through it.
+
+2. If you don't own the DLCs, you can download the missing files through [Mega.nz](#through-meganz) or [mirrors](#through-mirrors).
+
+### Through Mega.nz
+
+1. Download your language pack from [Mega.nz folder](https://mega.nz/folder/OTBWWAqY#KNyc53z1JPNUUWCS4qVqlQ).
+    - There's 3 options, downloading the ``full language folder``, ``without soundfiles`` and ``only DLC map``. 
+    - We recommend download the ``full language pack`` (~1GB) to get the original experience without any issues.
+
+### Through mirrors
+1. [https://h1.jokeruarwentto.tk/](https://h1.jokeruarwentto.tk/)
+    - Hosted in France, can be slow and full language pack only.
+
+## Installation
+
+<img align="right" src="/img/language-new-folder-result.png" />
+
+1. Into your ``game_root/zone`` directory, and create a new folder with your language name.
```

Comment:
"Navigate to Modern Warfare Remastered game folder and find your ``zone`` directory. Once there, create a new folder with the language name you are using (look below for reference)"